install.sh

```diff
@@ -372,6 +372,38 @@ get_release_version() {
     info "Using ${VERSION_K3S} as release"
 }
 
+# --- get k3s-selinux version ---
+get_k3s_selinux_version() {
+    available_version="k3s-selinux-1.2-2.${rpm_target}.noarch.rpm"
+    if [ "${INSTALL_K3S_SKIP_DOWNLOAD}" == true ]; then
```

Comment:
Maybe use the same set of tests as https://github.com/k3s-io/k3s/blob/4fdaa39d958c2ff926f44af5cea0e88fc1cc65b2/install.sh#L539

or just move the `get_k3s_selinux_version` call into the `else` case of that block?